mkdocs.yml

```diff
@@ -5,58 +5,89 @@ repo_url: https://github.com/ezsystems/user-documentation
+        - Content organization:
+            - Copy, move or hide content: content_management/content_organization/copy_move_hide_content.md
+            - Classify content: content_management/content_organization/classify_content.md
+            - Manage content locations and URLs: content_management/content_organization/manage_locations_urls.md
+    - Content publishing:
+        - Publish content: content_management/publish_instantly.md
+        - Schedule publication: content_management/schedule_publishing.md
+    - Search: 
+        - Search for content: search/search_for_content.md
+        - Search Engine Optimization: 
+            - SEO: search_engine_optimization/seo.md
+            - Work with SEO: search_engine_optimization/work_with_seo.md
     - Personalization:
-        - 'Personalization': 'personalization/personalization.md'
-        - 'Use cases': 'personalization/use_cases.md'
-        - 'Enable personalization': 'personalization/enabling_personalization.md'
-        - 'Configure personalization': 'personalization/perso_configuration.md'
-        - 'Preview scenario results': 'personalization/previewing_scenario.md'
-        - 'Integrate scenario results': 'personalization/integrating_results.md'
-        - 'Cross content type recommendations': 'personalization/cross_content_type_reco.md'
-        - 'Import data': 'personalization/content_import.md'
-        - 'Review performance': 'personalization/dashboard.md'
+        - Personalization: personalization/personalization.md
+        - Use cases: personalization/use_cases.md
+        - Enable personalization: personalization/enable_personalization.md
+        - Configure personalization: personalization/configure_personalization.md
+        - Preview scenario results: personalization/preview_scenario_results.md
+        - Integrate scenario results: personalization/integrate_scenario_results.md
+        - Cross content type recommendations: personalization/cross_content_type_reco.md
+        - Import data: personalization/content_import.md
+        - Review performance: personalization/review_perso_performance.md
         - Reference:
-              - 'Events': 'personalization/event_types.md'
-              - 'Content types': 'personalization/content_types.md'
-              - 'Recommendation models': 'personalization/recommendation_models.md'
-              - 'Scenarios': 'personalization/scenarios.md'
-              - 'Filters': 'personalization/filters.md'
+              - Events: personalization/event_types.md
+              - Content types: personalization/content_types.md
+              - Recommendation models: personalization/recommendation_models.md
+              - Scenarios: personalization/scenarios.md
+              - Filters: personalization/filters.md
+    - PIM:
+        - Products: pim/products.md
+        - Create product types: pim/create_product_types.md
+        - Add assets: pim/work_with_product_assets.md
```

Comment:
```suggestion
        - Work with product assets: pim/work_with_product_assets.md
```

Reply:
It  needs to be unified. The same as the doc's header. Add product assets, or Work with assets, as long as it's consistent.